.codeclimate.yml

```diff
@@ -1,4 +1,11 @@
 version: "2"
+plugins:
```

Comment:
Does this need to be under `plugin`? Does it work putting it under `checks`, as the other settings?

Reply:
Following the Configuring in https://github.com/codeclimate/codeclimate-duplication, it needs to be under plugin. We can use Code Climate CLI (https://github.com/codeclimate/codeclimate) to do analyze locally to see if it works as we want.